tests/operators/test_great_expectations.py

```diff
@@ -810,14 +812,64 @@ def test_great_expectations_operator__make_connection_string_mssql():
         port=5439,
     )
     operator.conn_type = operator.conn.conn_type
-    assert operator.make_connection_string() == test_conn_str
-
-
-def test_great_expectations_operator__make_connection_string_snowflake():
-    test_conn_str = (
-        "snowflake://user:password@account.region-east-1/"
-        "database/schema?warehouse=warehouse&role=role&private_key_file=/path/to/key.p8"
+    assert operator.make_connection_configuration() == test_conn_conf
+
+
+def test_great_expectations_operator__make_connection_string_snowflake(mocker):
+    test_conn_conf = {
+        "url": URL.create(
```

Comment:
@denimalpaca here is another reason why I used `url` instead of `connection_string`. The SQL Alchemy `URL` allows to build the exact same string which makes the test less brittle. I could of course change this to `{"connection_string": URL.create(...)}` if you want me to.